pkg/httphelper/server_test_utils.go

```diff
@@ -0,0 +1,115 @@
+package httphelper
+
+import (
+	"fmt"
+	"net"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"strconv"
+)
+
+var featuresReply = `{
+	"cassandra_version": "4.0.1",
+	"features": [
+		"async_sstable_tasks",
+		"rebuild"
+	]
+	}`
+
+var jobDetailsCompleted = `{"submit_time":"1638545895255","end_time":"1638545895255","id":"%s","type":"Cleanup","status":"COMPLETED"}`
```

Comment:
When I first saw this I wondered whether there should be a separate job details for the rebuild, but looking again at `reconcileEveryPodTask` I see that it only checks the status. Is `type` actually used anywhere?

Reply:
Not by cass-operator. But if one would have "getAllJobs" in the management-api, you could create external tool to monitor the jobs from pods.